app/views/planning_applications/_decision_notice.html.erb

```diff
@@ -12,27 +12,22 @@
     <strong><%= planning_application.decision.capitalize %></strong>
   </h2>
   <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
-  <dl class="govuk-summary-list govuk-summary-list--no-border">
-    <%
-      {
-        "Applicant" => planning_application.applicant_name,
-        "Application number" => planning_application.reference,
-        "Application received" => planning_application.created_at.to_formatted_s(:day_month_year),
-        "Decision date" => planning_application.determined_at&.to_formatted_s(:day_month_year) || "TBD",
-        "Site address" => planning_application.full_address,
-        "Use/development" => planning_application.description,
-      }.each do |key, value|
-    %>
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= key %>
-      </dt>
-      <dd class="govuk-summary-list__value">
```

Comment:
why did we move away from the definition list?

Reply:
design request